oem/firstvoices/windows/src/xml/config.js

```diff
@@ -279,9 +297,21 @@ function options_basekeyboard() {
 function keyboard_checkclick(n,toggle) {
   var k = document.getElementById('keyboardcheck_'+n), li = document.getElementById('list_keyboard_'+n);
   if(toggle) k.checked = !k.checked;
-  location.href='keyman:keyboard_clickcheck?id='+n+'&value='+k.checked;
+  location.href='keyman:keyboard_clickcheck?install=1&id='+n+'&value='+k.checked;
```

Comment:
Is `install=1` part of the support for enabling/disabling keyboards?

Reply:
Yes -- the FirstVoices Keyboards app installs all the keyboards but does not make them enabled until the user selects the ones they want to use.